### PR TITLE
fix: remove html tags to fix misalignment

### LINKED ISSUE #38156: Font misalignment on intro to managing npm packages
Screen Shot 2020-02-03 at 7 30 48 PM

For some reason, the font changes halfway down this introduction. [The file is here](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/client/src/pages/learn/apis-and-microservices/managing-packages-with-npm/index.md). I don't see the problem at first glance. Probably just a spacing issue or something.

### PR DESCRIPTION
**Pull request by vanya829**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #38156

### CODE REVIEW

File: client/src/pages/learn/apis-and-microservices/managing-packages-with-npm/index.md [hunk 1]
```diff
@@ -5,5 +5,13 @@ superBlock: APIs and Microservices
...
+
+npm saves packages in a folder named <code>node\_modules</code>. These packages can be installed in two ways:
+
+
+1. globally in a root <code>node\_modules</code> folder, accessible by all projects.
+2. locally within a project's own <code>node\_modules</code> folder, accessible only to that project.
+
+Most developers prefer to install packages local to each project to create a separation between the dependencies of different projects.
 Working on these challenges will involve you writing your code on Glitch on our starter project. After completing each challenge you can copy your public Glitch url (to the homepage of your app) into the challenge screen to test it! Optionally you may choose to write your project on another platform but it must be publicly visible for our testing.<br>Start this project on Glitch using <a href='https://glitch.com/edit/#!/remix/clone-from-repo?REPO_URL=https://github.com/freeCodeCamp/boilerplate-npm/'>this link</a> or clone <a href='https://github.com/freeCodeCamp/boilerplate-npm/'>this repository</a> on GitHub! If you use Glitch, remember to save the link to your project somewhere safe!
```
Comment (moT01) [inline client/src/pages/learn/apis-and-microservices/managing-packages-with-npm/index.md]:
```suggestion
Working on these challenges will involve you writing your code on Glitch on our starter project. After completing each challenge you can copy your public Glitch url (to the homepage of your app) into the challenge screen to test it! Optionally you may choose to write your project on another platform but it must be publicly visible for our testing.

Start this project on Glitch using <a href='https://glitch.com/edit/#!/remix/clone-from-repo?REPO_URL=https://github.com/freeCodeCamp/boilerplate-npm/'>this link</a> or clone <a href='https://github.com/freeCodeCamp/boilerplate-npm/'>this repository</a> on GitHub! If you use Glitch, remember to save the link to your project somewhere safe!
```

Comment (moT01) [reply client/src/pages/learn/apis-and-microservices/managing-packages-with-npm/index.md]: Can you add an empty line here so it looks like the rest of the intros. I think this suggestion should do it. Thanks.

**Review by moT01 — CHANGES_REQUESTED** [review]
One small suggestion. Perhaps slightly out of the scope, but easy to add here.

**Review by moT01 — APPROVED** [review]
Thanks @vanya829 - LGTM 🎉

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**